R/module_star_systems.R

```diff
@@ -28,9 +29,10 @@ star_systems_ui = function(id) {
         "column-gap" = "1rem"
       ),
       bslib::card(
-        bslib::card_header("Star System"),
+        bslib::card_header("System map"),
         bslib::card_body(
-          shiny::p("This is a placeholder for the star systems module content.")
+          style = "padding: 0; margin: 0; aspect-ratio: 1 / 1;",
```

Comment:
nitpick: Generally, this is perfectly fine, but lets keep the style across the codebase consistent and use `htmltools::css()` as we did in [module_star_systems.R](https://github.com/JKolomanski/REXoplanets/blob/f1151e79288e83eca733a864ad0f6f6bdf275f14/R/module_star_systems.R#L23C15-L23C29).
```suggestion
          style = htmltools::css(
            "padding" = "0",
            "margin" = "0",
            "aspect-ratio" = "1 /1"
          ),
```